lambda/__tests__/helpers/server.ts

```diff
@@ -14,13 +16,15 @@ const tryJSON = (str) => {
 
 const app = express();
 app.use(bodyParser.json());
-const router = express.Router();
+const appRouter = express.Router();
+const actionsRouter = express.Router();
+const apiRouter = express.Router();
 app.use(bodyParser.json());
 app.use(bodyParser.urlencoded({ extended: false }));
 app.set('etag', false);
 app.disable('x-powered-by');
 
-router.use(
+app.use(
```

Comment:
Why the change here?  Are the methods express.Router().use  and express().use the same funtionality wise?

Reply:
@thegentlemanphysicist that's just a rename and functionality does not change with it